com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/json/RestUtils.java

```diff
@@ -113,7 +113,7 @@ public static HashMap<String, ArrayList<String>> getIncludes(String tableName, S
 	public static Query getQuery(String tableName, String whereClause, List<Object> params) {
 		MTable table = getTable(tableName);
 
-		if (table != null && table.isView() && tableName.toLowerCase().endsWith("_vt")) {
+		if (table != null && table.isView() && table.getTableName().toLowerCase().endsWith("_vt")) {
```

Comment:
what's the goal of this change?
if tableName is null then the variable table is null and the condition is not reached